crypto/fipsmodule/service_indicator/service_indicator_test.cc

```diff
@@ -440,6 +547,117 @@ TEST(ServiceIndicatorTest, AESGCM) {
   ASSERT_EQ(approved, AWSLC_NOT_APPROVED);
 }
 
+TEST(ServiceIndicatorTest, ECDH) {
+  int approved = AWSLC_NOT_APPROVED;
+
+  bssl::UniquePtr<EC_GROUP> group(EC_GROUP_new_by_curve_name(NID_secp224r1));
```

Comment:
What about the other groups, and groups not approved?

Reply:
In another PR, can this test be parameterised to test all curves? Would be a lot of vectors, though and may not be needed. May be in that case, follow Torben's suggestion of making the test self-consistency rather than KATs.

Reply:
Groups other than the approved ones fail directly when using `EC_GROUP_new_by_curve_name` to create a `EC_GROUP`.
https://github.com/awslabs/aws-lc/blob/88d124deee4e948a78d8e710d35af5fc18259a82/include/openssl/ec.h#L104-L115